src/pages/Address/AddressComp.tsx

```diff
@@ -305,25 +312,58 @@ export const AddressTransactions = ({
   transactions,
   transactionsTotal: total,
   addressInfo: { addressHash },
+  timeOrderBy,
+  txTypeFilter,
 }: {
   address: string
   transactions: State.Transaction[]
   transactionsTotal: number
   addressInfo: State.Address
+  timeOrderBy: State.SortOrderTypes
+  txTypeFilter: TxTypeType
```

Comment:
These two properties are loaded from URL, why not import them from URL in the list component directly